stripe/util.py

```diff
@@ -116,11 +118,12 @@ def dashboard_link(request_id):
 
 def logfmt(props):
     def fmt(key, val):
-        val = str(val)
+        if not isinstance(val, basestring):
+            val = unicode(val)
```

Comment:
Mind annotating these lines with a comment describing why this is done? (I assume it's just to convert non-unicode strings to unicode, but it couldn't hurt because the naming of `basestring` is not intuitive.)